projects/packages/my-jetpack/_inc/components/onboarding-screen/connection-form/connection-form.module.scss

```diff
@@ -39,10 +42,10 @@
 	margin-left: -20px;
 	width: 100%;
 	color: var(--jp-gray-80, #2c3338);
-	font-size: 1rem;
+	font-size: 15px;
```

Comment:
Either we stick to (r)em or px. Mixing these two approaches and using rem just out of confidence misses the purpose I believe.

I think in My Jetpack we stick to px, we can re-evaluate that though!

Reply:
Other than the old code, we don't have rem anywhere else in the onboarding code 🤔